mars/services/cluster/api/web.py

```diff
@@ -333,3 +351,17 @@ async def get_node_thread_stacks(self, address: str) -> List[Dict]:
         path = f"{self._address}/api/cluster/stacks?address={address}"
         res = await self._request_url("GET", path)
         return list(json.loads(res.body)["stacks"])
+
+    async def get_node_log_content(self, size: int = None, address: str = None) -> str:
+        if size is None:
+            path = f"{self._address}/api/cluster/logs?address={address}"
+            res = await self._request_url("GET", path)
+            return str(json.loads(res.body)["content"])
+        elif size == -1:
+            path = f"{self._address}/api/cluster/logs?address={address}&&size={size}&&filename=test.log"
+            res = await self._request_url("GET", path)
+            return bytes.decode(res.body, encoding="utf8")
+        else:
+            path = f"{self._address}/api/cluster/logs?address={address}&&size={size}"
```

Comment:
For different arguments, only `path` is different, can we just generate different `path` and make the logic more unified?

Reply:
Fix.